Qt (for Windows) fails to build under LTO, due to multiple definition issues, i.e

multiple definition of `QAccessibleLineEdit::~QAccessibleLineEdit()';

Possibly related to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=94156.

diff --git a/qtbase/src/widgets/accessible/simplewidgets.cpp b/qtbase/src/widgets/accessible/simplewidgets.cpp
index 107fd729fe..0e61878f39 100644
--- a/qtbase/src/widgets/accessible/simplewidgets.cpp
+++ b/qtbase/src/widgets/accessible/simplewidgets.cpp
@@ -109,6 +109,8 @@ QString qt_accHotKey(const QString &text);
   \ingroup accessibility
 */
 
+QAccessibleLineEdit::~QAccessibleLineEdit(){};
+
 /*!
   Creates a QAccessibleButton object for \a w.
 */
diff --git a/qtbase/src/widgets/accessible/simplewidgets_p.h b/qtbase/src/widgets/accessible/simplewidgets_p.h
index 73572e3059..658da86143 100644
--- a/qtbase/src/widgets/accessible/simplewidgets_p.h
+++ b/qtbase/src/widgets/accessible/simplewidgets_p.h
@@ -155,6 +155,7 @@ class QAccessibleLineEdit : public QAccessibleWidget, public QAccessibleTextInte
 public:
     explicit QAccessibleLineEdit(QWidget *o, const QString &name = QString());
 
+    ~QAccessibleLineEdit();
     QString text(QAccessible::Text t) const override;
     void setText(QAccessible::Text t, const QString &text) override;
     QAccessible::State state() const override;
